tokio/src/runtime/handle.rs

```diff
@@ -604,20 +604,20 @@ enum TryCurrentErrorKind {
 
 impl fmt::Debug for TryCurrentErrorKind {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        use TryCurrentErrorKind::*;
+        use TryCurrentErrorKind as E;
         match self {
-            NoContext => f.write_str("NoContext"),
-            ThreadLocalDestroyed => f.write_str("ThreadLocalDestroyed"),
+            E::NoContext => f.write_str("NoContext"),
+            E::ThreadLocalDestroyed => f.write_str("ThreadLocalDestroyed"),
```

Comment:
Why E? Should this not be a K for kind?

I'm also okay with `TryCurrentErrorKind::` here.

Reply:
E means Eroor, and I may have been influenced by to golang :). I believe we in Rust the error have a kind, which can refer to: https://doc.rust-lang.org/std/io/enum.ErrorKind.html, and the use of `TryCurrentErrorKind ::` here is not tedious, so let us just use `TryCurrentErrorKind ::`.